google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudLibraryDependencyWriter.java

```diff
@@ -121,7 +123,13 @@ protected void run(@NotNull Result result) throws Throwable {
         List<MavenId> newMavenIds = mavenIdsMap.get(false);
         if (!newMavenIds.isEmpty()) {
           newMavenIds.forEach(
-              mavenId -> MavenDomUtil.createDomDependency(model, /* editor= */ null, mavenId));
+              mavenId -> {
+                MavenDomUtil.createDomDependency(model, /* editor= */ null, mavenId);
+                UsageTrackerProvider.getInstance()
+                    .trackEvent(GctTracking.CLIENT_LIBRARY_ADD_LIBRARY)
+                    .addMetadata(GctTracking.METADATA_LABEL_KEY, mavenId.getDisplayString())
```

Comment:
maybe add metadata to this to indicate the build system it came from (in this case its always "maven" but in the future we'll have others.